### PR TITLE
Feature/create sui coachmark demo

### PR DESCRIPTION
**Pull request by xmurcia**
## Category/Component

**TASK**:  https://jira.ets.mpi-internal.com/browse/MTR-71840

### Description, Motivation and Context

Create minimal demo to show the app tour with the default tooltip implemented in the molecule coachmark

### Types of changes

- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [X] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles
- [ ] 🛠️ Tool

### Screenshots - Animations

coachmark

### CODE REVIEW

File: components/molecule/coachmark/demo/index.js [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+        <Article className={`molecule-coachmark-section`}>
+          <H2>Props</H2>
+          <Paragraph>
+            All the props received by the coachmark component are the same
+            accepted by the react joyride library,{' '}
+            <a href="https://docs.react-joyride.com/" target="_blank">
+              please refer to the original documentation to get more details
+            </a>
+          </Paragraph>
```
Comment (jordevo) [inline components/molecule/coachmark/demo/index.js:24]: perhaps a string literal will help there? 

```suggestion
          <Paragraph>
            {`All the props received by the coachmark component are the same
            accepted by the react joyride library,
            <a href="https://docs.react-joyride.com/" target="_blank">
              please refer to the original documentation to get more details
            </a>`}
          </Paragraph>
```

**Review by jordevo — APPROVED** [review]

File: components/molecule/coachmark/demo/index.js (hunk 1 above)
Comment (xmurcia) [reply components/molecule/coachmark/demo/index.js:24]: It seems that the link is not rendered as a link in that way :(

**Comment by andresin87** [comment]
Are you sure the desired closing button is an atom-buton link?
https://github.com/SUI-Components/sui-components/blob/feature/create-sui-coachmark-demo/components/molecule/coachmark/src/Tooltip/index.js#L43

Can you confirm with the designer why not better configure it as design="flat" shape="circular"

**Comment by xmurcia** [comment]
> Are you sure the desired closing button is an atom-buton link? https://github.com/SUI-Components/sui-components/blob/feature/create-sui-coachmark-demo/components/molecule/coachmark/src/Tooltip/index.js#L43
> 
> Can you confirm with the designer why not better configure it as design="flat" shape="circular"

Sure I'll check with them

File: components/molecule/coachmark/demo/index.js (hunk 1 above)
Comment (jordevo) [reply components/molecule/coachmark/demo/index.js:24]: oh you're right! didn't think about the markup inside 😅

**Pull request merged**